packages/framework/src/Twig/MoneyExtension.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Twig;
+
+use Shopsys\FrameworkBundle\Component\Money\Money;
+use Twig_Extension;
+use Twig_SimpleFilter;
+
+class MoneyExtension extends Twig_Extension
+{
+    /**
+     * @return array
+     */
+    public function getFilters()
+    {
+        return [
+            new Twig_SimpleFilter(
+                'moneyFormat',
+                [$this, 'moneyFormatFilter']
+            ),
+        ];
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Money\Money $money
+     * @param int|null $decimal
+     * @param string $decimalPoint
+     * @param string $thousandsSeparator
+     * @return string
+     */
+    public function moneyFormatFilter(Money $money, int $decimal = null, string $decimalPoint = '.', string $thousandsSeparator = '')
+    {
+        $moneyString = $money->toString();
+
+        if ($decimal === null) {
+            $decimal = $this->getNumberOfDecimalPlaces($moneyString);
```

Comment:
The usage `$decimal = strlen($money->centsOnly()->toValue()) - 2;` would be a bit cumbersome, because you would have to check `->isZero()` as well, otherwise you could get -1 (`strlen('0') - 2`).

We consider the method as not so important as to include it now.